inlang.config.js

```diff
@@ -0,0 +1,18 @@
+export async function defineConfig(env) {
+    const { default: i18nextPlugin } = await env.$import(
+      "https://cdn.jsdelivr.net/npm/@inlang/plugin-i18next@2/dist/index.js"
```

Comment:
Can this be done in a way, that it would also work offline? / Doesn't change without us knowing?

Reply:
I could add the plugin to the repo. Just keep in mind that you can access updates automatically.

Reply:
Ah I thought that you want it completely isolated in the repo. Added the package.